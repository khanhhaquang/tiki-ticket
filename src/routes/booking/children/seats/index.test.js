import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import Seats from './index';

describe('Test Seats component', () => {
	test('Snapshot seats', () => {
		const component = renderer.create(<Seats />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});

	const component = mount(<Seats />);
	test('Check each row has name and inner', () => {
		expect(component.find('.row.name')).toBeTruthy();
		expect(component.find('.row.inner')).toBeTruthy();
	});
});
