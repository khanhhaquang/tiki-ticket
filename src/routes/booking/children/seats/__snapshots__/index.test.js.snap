// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Seats component Snapshot seats 1`] = `
<main
  className="sc-bwzfXH jrgzpK"
>
  <div
    className="row"
  >
    <span
      className="name"
    >
      A
    </span>
    <div
      className="inner"
    >
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          1
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          2
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          3
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          4
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          5
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          6
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          7
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          8
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          9
        </span>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <span
      className="name"
    >
      B
    </span>
    <div
      className="inner"
    >
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          1
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          2
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          3
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          4
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          5
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          6
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          7
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          8
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          9
        </span>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <span
      className="name"
    >
      C
    </span>
    <div
      className="inner"
    >
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          1
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          2
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          3
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          4
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          5
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          6
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          7
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          8
        </span>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <span
      className="name"
    >
      D
    </span>
    <div
      className="inner"
    >
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          1
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          2
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat standard"
        onClick={[Function]}
      >
        <span>
          3
        </span>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <span
      className="name"
    >
      E
    </span>
    <div
      className="inner"
    >
      <div
        className="sc-bdVaJa bCssIi seat vip"
        onClick={[Function]}
      >
        <span>
          1
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat vip"
        onClick={[Function]}
      >
        <span>
          2
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat vip"
        onClick={[Function]}
      >
        <span>
          3
        </span>
      </div>
      <div
        className="sc-bdVaJa bCssIi seat vip"
        onClick={[Function]}
      >
        <span>
          4
        </span>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <span
      className="name"
    >
      F
    </span>
    <div
      className="inner"
    >
      <div
        className="sc-bdVaJa bCssIi seat vip"
        onClick={[Function]}
      >
        <span>
          1
        </span>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <span
      className="name"
    >
      G
    </span>
    <div
      className="inner"
    >
      <div
        className="sc-bdVaJa bCssIi seat vip"
        onClick={[Function]}
      >
        <span>
          1
        </span>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <span
      className="name"
    >
      H
    </span>
    <div
      className="inner"
    >
      <div
        className="sc-bdVaJa bCssIi seat vip"
        onClick={[Function]}
      >
        <span>
          1
        </span>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <span
      className="name"
    >
      I
    </span>
    <div
      className="inner"
    >
      <div
        className="sc-bdVaJa bCssIi seat vip"
        onClick={[Function]}
      >
        <span>
          1
        </span>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <span
      className="name"
    >
      J
    </span>
    <div
      className="inner"
    >
      <div
        className="sc-bdVaJa bCssIi seat vip"
        onClick={[Function]}
      >
        <span>
          1
        </span>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <span
      className="name"
    >
      K
    </span>
    <div
      className="inner"
    >
      <div
        className="sc-bdVaJa bCssIi seat deluxe"
        onClick={[Function]}
      >
        <span>
          1
        </span>
      </div>
    </div>
  </div>
</main>
`;
